cmd/ocp.go

```diff
@@ -94,7 +94,7 @@ func openShiftCmd() *cobra.Command {
 			envVars["ALERTS"] = ""
 		}
 		// If metricsEndpoint is not set, use values from flags
-		if workloadConfig.MetricsEndpoint == "" {
+		if workloadConfig.MetricsEndpoint == "" && esServer != "" && esIndex != "" {
```

Comment:
Bug?

Reply:
Its not exactly a bug, but an additional check to prevent overrides.